requirements.txt

```diff
@@ -1,2 +1,6 @@
+mkdocs
```

Comment:
I think the GitHub action uses requirements.txt to install plugins - I don't think this can be used here - but you might find it works

Reply:
I guess it is using your requirements.txt.  That's the 'else'.
So this will override the mkdocs tool that is in action's environment.

Reply:
maybe we need a mkdocs_requirements.txt?

Reply:
mkdocs_requirements.txt would be less ugly than listing the requirements in the readme and expecting folks to install them based on that.